openapi/SwarmCommon.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 3.2.7
+  version: 3.2.8
```

Comment:
The API has breaking changes (renaming two fields) and the version is only updating the bugfix number. To be semantically correct, API number should be raised to 4, or to keep old fields for backward compatibility. It depends on the expectations for the API stability, but since the major version is not 0, the API supposed to be stable.